sphinx/tutorial/rose/suites.rst

```diff
@@ -469,7 +471,6 @@ Rose Applications In Rose Suite Configurations
          [env]
          INTERVAL=60
          N_FORECASTS=5
-         WEIGHTING=1
```

Comment:
This was removed in an earlier step:

https://github.com/metomi/rose/blob/2117df70812b24b14cd7d455832e26f2321ea53a/sphinx/tutorial/rose/suites.rst?plain=1#L402-L406